### PR TITLE
Add field component timepicker

### PR DESCRIPTION
**Pull request by agungid**
- require package clockpicker
- Add time field component

### CODE REVIEW

File: resources/views/components/fields/timepicker.blade.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div class="form-group{{ $errors->has($field['name']) ? ' has-error' : '' }}">
+  <label class="control-label">{{ !empty($field['label']) ? $field['label'] : title_case(str_replace('_', ' ', snake_case($field['name']))) }}{{ !empty($field['required']) ? '*' : '' }}</label>
+  <div class="input-group time">
```
Comment (frdteknikelektro) [inline resources/views/components/fields/timepicker.blade.php]: Could you change it to
```
<div class="input-group js-clockpicker">
```

File: resources/assets/js/inspinia.js [hunk 2]
```diff
@@ -85,6 +88,9 @@ $(document).ready(function() {
   $('.js-datepicker').datepicker({
     todayHighlight: true
   });
+  $('.time').clockpicker({
```
Comment (frdteknikelektro) [inline resources/assets/js/inspinia.js]: Could you change it to
```
$('.js-clockpicker').clockpicker({
```

File: resources/views/components/fields/timepicker.blade.php (hunk 1 above)
Comment (frdteknikelektro) [reply resources/views/components/fields/timepicker.blade.php]: And the file also clockpicker.blade.php

**Comment by frdteknikelektro** [comment]
@agungid I've add some comment before to accept your PR.
please consider to modified it, and don't forget to npm run production.

Thank you btw. :+1:

**Comment by frdteknikelektro** [comment]
And could you also update readme.md ?

**Comment by agungid** [comment]
@frdteknikelektro Thanks. Maybe it could be another time. Now there is still busy. Hehe

**Comment by frdteknikelektro** [comment]
I've check it, 
Will be included on next release.
:tada:

**Comment by frdteknikelektro** [comment]
Check this out [v0.4.2](https://github.com/Atnic/laravel-inspinia/releases/tag/v0.4.2)

**Pull request merged**